lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -118,7 +126,7 @@ def parse(body)
 
       def build_xml_request
         builder = Nokogiri::XML::Builder.new(encoding: 'UTF-8') do |xml|
-          xml.transactionWithCard 'xmlns' => 'http://secure.ixopay.com/Schema/V2/TransactionWithCard' do
+          xml.transactionWithCard 'xmlns' => 'https://secure.cardflo.io/Schema/V2/TransactionWithCard' do
```

Comment:
can we define this like:
`self.transaction_with_card_url = 'https://secure.cardflo.io/Schema/V2/TransactionWithCard'`
on top of class, like test_url and live_url.